app/templates/gentelella/admin/base.html

```diff
@@ -5,7 +5,7 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>{% block title %}{% endblock %} - {{ app_name }}</title>
+    <title>{{ app_name }} - {% block title %}{% endblock %}</title>
```

Comment:
@aviaryan Yes, this will become open Event-manage Events etc. I tried to refer to eventbrite site. They had it that way. So I changed it like that.
![screenshot from 2017-01-14 18-34-23](https://cloud.githubusercontent.com/assets/20799954/21955044/58aa0b42-da88-11e6-9f65-ed8d0d3c5c64.png)